drivers/npmx/npmx_shell.c

```diff
@@ -695,7 +721,13 @@ static int buck_voltage_get(const struct shell *shell, size_t argc, char **argv,
 	}
 
 	if (check_error_code(shell, err_code)) {
-		shell_print(shell, "Value: %d mV.", npmx_buck_voltage_convert_to_mv(voltage));
+		uint32_t voltage_mv;
+
+		if (npmx_buck_voltage_convert_to_mv(voltage, &voltage_mv)) {
+			shell_print(shell, "Value: %d mV.", voltage_mv);
+		} else {
+			shell_error(shell, "Error: unable to convert buck voltage value to mV.");
```

Comment:
Could You please change 'buck' to 'Buck'?

Reply:
We use lowercase throughout the file, any reason to change this?

Reply:
I thought there was a rule to always use 'Buck' in error messages.
But now I see that 'Buck' is used only if it's first word in an error msg.
https://projecttools.nordicsemi.no/bitbucket/projects/NCS-TEST/repos/test-sdk-npmx/browse/bts/cli/npmx_cli_base.py#189,192-194,198,200-201